_test/test_data_loaders/test_asciipar_loader.m

```diff
@@ -81,7 +81,7 @@ function test_load_ASCII_par_binary(this)
             assertTrue(all(ismember({'filename','filepath','x2','phi','azim','width','height','group'},fields(par))));
             assertTrue(all(ismember(fields(par),{'filename','filepath','x2','phi','azim','width','height','group'})));
             assertEqual(28160,numel(par.x2))
-            assertEqual(28160,al.n_detectors)
+            assertEqual(28160,al.n_det_in_par)
```

Comment:
where does the magic number `28160` come from? it appears in lots of the tests

Reply:
Its the number of detectors and it sits in the data file, used for testing. there were 2 such numbers for two files, but it should go to one now, as the bigger one does not fit github -- ASCII is too big.

Reply:
If there are data files too large for GitHub they can go into the new network storage when we get that configured

Reply:
Can that one number be extracted as a constant (`EXPECTED_NUM_DETECTORS`) or something since it's repeated so many time and matches up to a static value in the data file?

Reply:
Not sure it improves readability, but as I am making changes anyway, will change this too.

Reply:
Thanks Alex -- the advantage to me is that it explains what that number represents and reduces the likelihood of typos in typing the same 5-digit number lots of time